File: SimpleInterest.hs
calculateSimpleInterest :: Float -> Float -> Float -> Float
calculateSimpleInterest principal rate time = (principal * rate * time) / 100

main :: IO ()
main = do
    let principal = 1000
    let rate = 5
    let time = 3
    let interest = calculateSimpleInterest principal rate time
    putStrLn $ "The simple interest is: " ++ show interest

File: simple_interest/simple_interest.erl
-module(simple_interest).
-export([calculate_simple_interest/3, main/0]).

calculate_simple_interest(Principal, Rate, Time) ->
    (Principal * Rate * Time) / 100.

main() ->
    Principal = 1000,
    Rate = 5,
    Time = 3,
    Interest = calculate_simple_interest(Principal, Rate, Time),
    io:format("The simple interest is: ~p~n", [Interest]).
